src/EventStore.Client.Streams/EventStoreClient.Subscriptions.cs

```diff
@@ -6,33 +6,10 @@
 #nullable enable
 namespace EventStore.Client {
 	public partial class EventStoreClient {
-		private Task<StreamSubscription> SubscribeToAllAsync(
```

Comment:
**QUESTION:** Do we want to remove previous methods entirely? Maybe it'd be worth making them obsolete and giving people a chance to migrate?